beacon_node/store/src/lib.rs

```diff
@@ -174,8 +187,12 @@ pub fn parse_data_column_key(data: Vec<u8>) -> Result<(Hash256, ColumnIndex), Er
 #[must_use]
 #[derive(Clone)]
 pub enum KeyValueStoreOp {
-    PutKeyValue(Vec<u8>, Vec<u8>),
-    DeleteKey(Vec<u8>),
+    // Indicate that a PUT operation should be made
+    // to the db store for a (Column, Key, Value)
+    PutKeyValue(String, Vec<u8>, Vec<u8>),
+    // Indicate that a DELETE operation should be made
+    // to the db store for a (Column, Key)
+    DeleteKey(String, Vec<u8>),
```

Comment:
I think we should use `DBColumn` here directly, or failing that `&'static str`. `DBColumn` has the advantage of being type safe _and_ saving heap allocations, while `&'static str` just saves heap allocations.

Reply:
This commit moves to use `DBColumn` everywhere I've found to be appropriate [6dfd984](https://github.com/sigp/lighthouse/pull/4718/commits/6dfd9843ca83af3ca720a1196b3d84e489e8e175)